lib/IRGen/GenMeta.cpp

```diff
@@ -1071,15 +1071,13 @@ namespace {
             SILDeclRef func(entry.getFunction());
 
             /// Distributed thunks don't need resilience.
-            if (func.isDistributedThunk()) {
-              continue;
+            if (!func.isDistributedThunk()) {
```

Comment:
Nit: please fix the comment to talk about method descriptors.

Reply:
Gotcha, will do - seems this broke on TBD test I just noticed, so more work here to be done anyway still. I'll ping when updated 🫡